Ginger/GingerCore/Drivers/JavaDriverLib/JavaDriver.cs

```diff
@@ -2860,6 +2861,11 @@ private void DoRecordings(bool isPomRecord = false)
                     Value = ""
                 });
 
+                if(isPOMRecording)
+                {
+                    parentBrowserPath = ci.XPath;
```

Comment:
I think this should be done using Browser Changed Event in Recording manager, Like we have page changed event